src/TLuaInterpreter.cpp

```diff
@@ -5869,6 +5869,37 @@ int TLuaInterpreter::tempButton(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::setButtonStyleSheet(lua_State* L)
+{
+    //args: name, css text
+    QString name, css;
+
+    if (!lua_isstring(L, 1)) {
+        lua_pushstring(L, "setButtonStyleSheet: wrong first arg");
+        lua_error(L);
+        return 1;
+    } else {
+        name = lua_tostring(L, 1);
+    }
+    if (!lua_isstring(L, 2)) {
+        lua_pushstring(L, "setButtonStyleSheet: wrong second arg");
+        lua_error(L);
+        return 1;
+    } else {
+        css = lua_tostring(L, 2);
+    }
+
+    Host& host = getHostFromLua(L);
+    std::list<TAction*> actionsList = host.getActionUnit()->findActionsByName(name);
+    if (actionsList.empty()) {
+        return 0;
```

Comment:
It would be a good idea to return `nil` to Lua here so if a script wants to, they can know whenever their command worked or not. ie:

```
        lua_pushnil(L);
        lua_pushfstring(L, "setButtonStyleSheet: no button named \"%s\" found", name);
        return 2;
```